.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: close needs-information issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          days-before-issue-stale: 14
```

Comment:
I think it is possible to use `-1` here, and then change `days-before-issue-close` to `14` or `21` for the issue to be closed straight away, without receiving a `stale` label.